### DISCUSSION TITLE
training with existing graph.pb

### DISCUSSION THREAD

**Discussion by evenfarther** [deepmodeling/deepmd-kit#3322] [Q&A]
Dear DeepMD-kit developers.

Hello! thank you for providing the wonderful code!

I have a simple question.

i want to train a new model with pre-trained graph.pb.

so, i use 'dp train -f graph.pb' to train a new model.

after training is completed, i freeze the model using 'dp freeze -o graph.pb'

then, i compress the model by 'dp compress -i graph.pb -o graph_compressed.pb -t input.json'.

however, this error message occured.

RuntimeError: The input frozen model has already been compressed! Please do not compress the model repeatedly.

1. how can i handle above error message?
2. is it okay to run lammps MD based on graph.pb, not graph_compressed.pb?

**Comment by njzjz** [comment]
I guess the pre-trained `graph.pb` has already been compressed. In this case, you trained the model against the compressed descriptor. (Sorry, this feature seems to be not documented.)
